tools/deployment/package/docker-compose-all.yaml

```diff
@@ -486,3 +486,30 @@ services:
         "-f",
         "http://mcp_server:8000/health"
       ]
+
+  api-server:
```

Comment:
sorry for missing this in the code review - a healthcheck should be added to check the liveness of the service, as done in the mcp-server

@coderabbitai create an issue to check